css/css-ui/webkit-appearance-parsing.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Parsing of `-webkit-appearance`</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<script>
+test(function() {
+  var input = document.createElement('input');
+  input.style = '-webkit-appearance: none;';
```

Comment:
This needs to use `setAttribute()` instead. The setter here is forwarded to the `cssText` setter.